### PR TITLE
Added new Security Quiz Questions

### PR DESCRIPTION
**Pull request by nitesh2920**
## Summary of changes

Added three new question related to security from the freecodecamp articles.
## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/security-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,31 @@
 const securityQuiz = [
+  {
+    Question:"What is the primary purpose of a wordlist in ethical hacking ?",
+    Answer : "To automate password cracking attempts",
+    Distractor1: "To store user credentials securley",
```
Comment (a2937) [inline src/data/security-quiz.ts:5]: You spelled "securely" wrong.

Comment (nitesh2920) [reply src/data/security-quiz.ts:5]: Okay, I will make the required changes. Is there anything else? @a2937

Comment (nitesh2920) [reply src/data/security-quiz.ts]: I had made the required changes please review it.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**